net/quagga/src/opnsense/mvc/app/models/OPNsense/Quagga/Menu/Menu.xml

```diff
@@ -1,6 +1,6 @@
 <menu>
   <Routing cssClass="fa fa-map-signs" order="45">
-    <General VisibleName="General" cssClass="fa fa-map-signs fa-fw" url="/ui/quagga/general/index" order="1"/>
+    <General VisibleName="General" cssClass="fa fa-cogs fa-fw" url="/ui/quagga/general/index" order="1"/>
```

Comment:
Is this a single item or the multiple ones? For settings pages we use the single one